numpy/f2py/diagnose.py

```diff
@@ -35,11 +35,11 @@ def run():
         has_newnumpy = 0
 
     try:
-        from numpy.f2py import f2py2e
-        has_f2py2e = 1
+        from numpy.f2py import f2pyarg
+        has_f2pyarg = 1
     except ImportError:
-        print('Failed to import f2py2e:', sys.exc_info()[1])
-        has_f2py2e = 0
+        print('Failed to import f2pyarg:', sys.exc_info()[1])
+        has_f2pyarg = 0
```

Comment:
@pearu suggested that we can and should shift from `0` and `1` to `True` and `False`.

Reply:
Yes, I have made changes in f2pyarg, will do here too.